docs/vspec2proto.md

```diff
@@ -0,0 +1,98 @@
+# vspec2proto - vspec Protocol Buffer generator
+
+The vspec2proto.py script generates [Protocol Buffer message definitions](https://protobuf.dev) for all nodes in the tree. You can use these proto definitions to serialize the data of a VSS tree, e.g., as part of a gRPC API.
+
+## Example
+
+```bash
+./vspec2protobuf.py outputIds_v2.vspec indentity.proto  -q ../vehicle_signal_specification/spec/quantities.yaml -u ../vehicle_signal_specification/spec/units.yaml  --static-uid --add-optional
```

Comment:
Do you also get warnings like:

`WARNING  Attribute(s) staticUID in element Major not a core or known extended attribute.`

In https://github.com/COVESA/vss-tools/blob/master/docs/vspec2id.md#generate-eg-yaml-file-with-static-uids they include `-e staticUID` in examples to avoid warnings

I am thinking if we promote `staticUID` to a `core_attributes` so that you never get a warning. Thoughts @adobekan

Reply:
Yes, I do get these warnings too. I thought this would be "normal" since `staticUID` is not a `core_attributes`. So thanks for the hint of how to avoid the warnings. I will have a look into it.

Reply:
Short term I think you could just add `-e staticUID` in the example. But I am thinking of the long term solution where I see multiple alternatives:

- Promote `staticUID` as an allowed/core attribute for all tools always
- Add it in some way as core-attribute for the specific tool, regardless of whether `--static-uid` is used or not
- Add it only if  `--static-uid` is used

For the last two alternatives we should find a common mechanism that can reused by all tools.  I anyway not see it as a topic for this PR.

Reply:
The documentation now contains an example with the parameter `-e staticUID`.